flask_meld/meld.py

```diff
@@ -0,0 +1,44 @@
+import os
+from flask import send_from_directory
+from flask_socketio import SocketIO
+from .tag import MeldTag, MeldScriptsTag
+from .message import process_message
+
+
+class Meld:
+    def __init__(self, app=None):
+        self.app = app
+
+        if app is not None:
+            self.init_app(app)
```

Comment:
if you use `self.app` you don't have to pass it to this method, you could use `self.app` there